### PR TITLE
Dev -> Master for v1.0 release

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,14 +3,27 @@
...
+
+| `SRA`        | `ENA`        | `GEO`      |
+|--------------|--------------|------------|
+| SRR11605097  | ERR4007730   | GSM4432381 |
+| SRX8171613   | ERX4009132   | GSE147507  |
+| SRS6531847   | ERS4399630   |            |
+| SAMN14689442 | SAMEA6638373 |            |
+| SRP256957    | ERP120836    |            |
+| SRA1068758   | ERA2420837   |            |
+| PRJNA625551  | PRJEB37513   |            |
```
Comment (maxulysse) [inline CHANGELOG.md:29]: I'm guessing that the IDs here are just examples, and not the only supported IDs from the DB

Comment (drpatelh) [reply CHANGELOG.md:29]: Yep. Just examples.

Comment (drpatelh) [reply CHANGELOG.md:29]: Re-worded https://github.com/nf-core/fetchfastq/pull/3/commits/7fb57784448a7a8f29b8beb89e86b6ea48f50013

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -74,10 +74,8 @@ If you wish to contribute a new step, please use the following coding standards:
 7. Add sanity checks for all relevant parameters.
 8. Add any new software to the `scrape_software_versions.py` script in `bin/` and the version command to the `scrape_software_versions` process in `main.nf`.
 9. Do local tests that the new code works properly and as expected.
-10. Add a new test command in `.github/workflow/ci.yaml`.
-11. If applicable add a [MultiQC](https://https://multiqc.info/) module.
-12. Update MultiQC config `assets/multiqc_config.yaml` so relevant suffixes, name clean up, General Statistics Table column order, and module figures are in the right order.
-13. Optional: Add any descriptions of MultiQC report sections and output files to `docs/output.md`.
+10. Add a new test command in `.github/workflow/ci.yml`.
+11. Optional: Add any descriptions of MultiQC report sections and output files to `docs/output.md`.
```
Comment (JoseEspinosa) [inline .github/CONTRIBUTING.md:78]: Is the 11 point needed? The pipeline has not MultiQC report, has it?

File: README.md [hunk 3]
```diff
@@ -16,20 +16,22 @@
...
 
-<!-- TODO nf-core: Add full-sized test dataset and amend the paragraph below if applicable -->
 On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/fetchfastq/results).
 
 ## Pipeline summary
 
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
+Via a single file of ids, provided one-per-line the pipeline performs the following steps:
```
Comment (JoseEspinosa) [inline README.md:27]:
```suggestion
Via a single file of ids, provided one-per-line, see an example [here](https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/public_database_ids.txt), the pipeline performs the following steps:
```

File: lib/WorkflowFetchfastq.groovy [hunk 4]
```diff
@@ -7,53 +7,26 @@ class WorkflowFetchfastq {
...
-        }
+    public static void curateSamplesheetWarn(log) {
+        log.warn "=============================================================================\n" +
+            "  Please double-check the samplesheet that has been auto-created by the pipeline.\n\n" +
+            "  Public databases don't reliably hold information such as strandedness\n" +
+            "  information, controls etc\n\n" +
+            "  All of the sample metadata obtained from the ENA has been appended\n" +
+            "  as additional columns to help you manually curate the samplesheet before\n" +
+            "  running nf-core/other pipelines.\n" +
+            "==================================================================================="
```
Comment (JoseEspinosa) [inline lib/WorkflowFetchfastq.groovy:30]: I love it ❤️

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (JoseEspinosa) [inline .github/CONTRIBUTING.md:78]:
```suggestion
```

**Review by JoseEspinosa — APPROVED** [review]
Small comments! Very nice work @drpatelh! 😍

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (drpatelh) [reply .github/CONTRIBUTING.md:78]: Fixed https://github.com/nf-core/fetchfastq/pull/5/commits/5a488f5d6714633830777afb75d047332c12bbb6

Comment (drpatelh) [reply .github/CONTRIBUTING.md:78]: Fixed https://github.com/nf-core/fetchfastq/pull/5/commits/5a488f5d6714633830777afb75d047332c12bbb6

File: README.md (hunk 3 above)
Comment (drpatelh) [reply README.md:27]: Fixed https://github.com/nf-core/fetchfastq/pull/5/commits/70328a959324f3abb6b03c2dc6396527dbb3adb3

**Review by Emiller88 — APPROVED** [review]

**Comment by drpatelh** [comment]
Thanks guys!!

**Pull request merged**